test/data/valid_single_bundle.tf

```diff
@@ -0,0 +1,33 @@
+provider "instaclustr" {
+  username = "%s"
+  api_key = "%s"
+  api_hostname = "%s"
+}
+
+resource "instaclustr_cluster" "valid" {
+  cluster_name = "testcluster"
+  node_size = "t3.small"
+  data_centre = "US_WEST_2"
+  sla_tier = "NON_PRODUCTION"
+  cluster_network = "192.168.0.0/18"
+  private_network_cluster = false
+  pci_compliant_cluster = false
+  cluster_provider = {
+    name = "AWS_VPC"
+  }
+  rack_allocation = {
+    number_of_racks = 3
+    nodes_per_rack = 1
```

Comment:
A non-default rack allocation would be good to have instead. This would give you confidence that it handles not just default behaviour but also custom values E.g: 4 racks and 2 nodes per rack

Reply:
4 racks not accepted as we should pick one from [1, 3, 5];

so I updated it to 

number_of_racks = 5
 nodes_per_rack = 2